core/client/pom.xml

```diff
@@ -67,47 +67,6 @@
       <artifactId>alluxio-core-protobuf</artifactId>
       <version>${project.version}</version>
     </dependency>
-    <dependency>
-      <groupId>org.alluxio</groupId>
-      <artifactId>alluxio-underfs-gcs</artifactId>
-      <version>${project.version}</version>
-    </dependency>
-    <!--Add Alluxio underfs modules so by including the client uber jar, application can talk to different underfs.-->
```

Comment:
shall we transfer this comment to pom of `alluxio-underfs-glusterfs`? I don't want to loss this information, looks useful to me.